packages/cli/src.ts/bin/ethers.ts

```diff
@@ -498,7 +498,7 @@ class SweepPlugin extends Plugin {
         await this.accounts[0].sendTransaction({
             to: this.toAddress,
             gasLimit: 21000,
-            gasPrice: gasPrice,
+            // gasPrice: gasPrice,
```

Comment:
Instead of doing small changes/comments in a lot of places in the code, lets remove the compilation for the elements that we do not support, such as the CLI and the hardware wallets.

Reply:
I suggest we remove the `cli` and the hardware wallets packages in a separate PR.